python/pyiceberg/expressions/literals.py

```diff
@@ -491,3 +531,8 @@ def _(self, type_var: FixedType) -> Optional[Literal[bytes]]:
             return FixedLiteral(self.value)
         else:
             return None
+
+
+def _transform_literal(func: Callable[[T], T], lit: Literal[T]) -> Literal[T]:
+    """Small helper to unwrap the value from the literal, and wrap it again"""
+    return literal(func(lit.value))
```

Comment:
I'd probably move this over to transforms since it is only used there.

Reply:
Good one, done! 👍🏻